mpi-proxy-split/Makefile_config.in

```diff
@@ -37,7 +37,7 @@ else
   MPICXX = mpic++ -std=c++14
   MPIFORTRAN = mpifort
   MPIRUN = mpirun
-  MPI_LD_FLAG = -L$$HOME/mpich-static/usr/lib64 -lmpi -llzma -lz -lm -lxml2
+  MPI_LD_FLAG = -L$$HOME/mpich-static/usr/lib64 -lmpi -L$$HOME/local_install/lib -llzma -lz -lm -lxml2
```

Comment:
I suggest a FIXME comment above this line, to indicate that $$HOME/local_install/lib is a hard-wired pathname, and we should eventually provide a way to configure this pathame in configure-mana.

QUESTION: Are liblzma.a, libz.a, libm.a and libxml2.a the only extra libraries needed by static MPICH?  This doesn't affect the current PR, but I want to update the documentation in lower-half/README.mpich-static.  Thanks.

Reply:
Added a FIXME.

Yes, these are the only libraries required on discovery when using the default mpich module (i.e., `mpich/3.3.2`).